tasklauncher/spring-cloud-starter-stream-sink-tasklauncher-cloudfoundry/src/main/resources/META-INF/spring-configuration-metadata-whitelist.properties

```diff
@@ -0,0 +1 @@
+configuration-properties.classes=
```

Comment:
1) No, these are just app properties that happen to be overridable _via_ deployment properties. Your example in the README could has well have used `... | cftask --deployer.url=https://api.local.pcfdev.io` _etc_.
2) I don't understand what you're saying, but in any case, there should only be just one set of props now, the former.